src/style.scss

```diff
@@ -0,0 +1,136 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding-top: 32px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+
+  &__photo {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__name {
+    padding: 0 16px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    color: #060b35;
+  }
+
+  &__code {
+    padding: 0 16px;
+    margin-top: 4px;
+    font-weight: 400;
```

Comment:
font-weight: 400; is a default browser value. So you can delete at similar cases as well
```suggestion
```